src/Generated/Drives/Item/Items/Item/Workbook/Functions/Acot/AcotPostRequestBody.php

```diff
@@ -63,28 +62,14 @@ public function getBackingStore(): BackingStore {
     public function getFieldDeserializers(): array {
         $o = $this;
         return  [
-            'number' => fn(ParseNode $n) => $o->setNumber($n->getObjectValue([Json::class, 'createFromDiscriminatorValue'])),
         ];
     }
 
-    /**
-     * Gets the number property value. The number property
-     * @return Json|null
-    */
-    public function getNumber(): ?Json {
```

Comment:
Looks like a breaking change plus a couple of other changes relating to UntypedJson types.
cc @andrueastman , @petrhollayms